src/styles/blocks/card.scss

```
@@ -0,0 +1,128 @@
+/* #region card */
+.card {
+  box-sizing: border-box;
+  flex-direction: column;
+  margin: 0;
+  width: $card-width;
+  border: $border-width solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
```

here and below always group modificator's
see [recommendations](http://codeguide.academy/html-css.html#css-order)
```suggestion
  flex-direction: column;
  
  box-sizing: border-box;
  width: $card-width;
  margin: 0;
  
  border: $border-width solid rgba(243, 243, 243, 1);
  border-radius: 5px;
```